docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -15,19 +15,17 @@ You have a stream of events that represent a series of changes, known as a chang
 ## In action
 
 ```sql
-CREATE TABLE t1 AS
-    SELECT k,
-           LATEST_BY_OFFSET(v1) AS v1,
-           LATEST_BY_OFFSET(v2) AS v2,
-           LATEST_BY_OFFSET(v3) AS v3
+CREATE SOURCE TABLE t1 AS
+    SELECT k, v1, v2, v3
     FROM s1
     GROUP BY k
     EMIT CHANGES;
 ```
-
 ## Materializing a changelog stream
 
-In ksqlDB, you derive new tables by aggregating other streams and tables. To create a table that reflects the latest values for each key, use the `LATEST_BY_OFFSET` aggregation.
+In ksqlDB, you derive new tables by aggregating other streams and tables.
+To create a table that reflects the latest values for each key, use the 
+CREATE SOURCE TABLE statement.
```

Comment:
This one becomes a little tricky to rewrite... If we do a `CREATE SOURCE TABLE`, we don't allow to use `INSERT INTO` because a "source table" is read-only.

We could keep the CS statement as-is an insert into the stream. And only later replace the `CTAS` statement with `CREATE SOURCE TABLE`. Not sure if this would convey the original intent of this example? \cc @MichaelDrogalis